src/arithmetic/numeric_funcs/numeric_funcs.c

```diff
@@ -202,5 +212,10 @@ void Register_NumericFuncs() {
 	array_append(types, (SI_NUMERIC | T_NULL));
 	func_desc = AR_FuncDescNew("sqrt", AR_SQRT, 1, 1, types, true, false);
 	AR_RegFunc(func_desc);
+
+	types = array_new(SIType, 1);
+	array_append(types, (SI_NUMERIC | T_NULL));
```

Comment:
this function takes 2 arguments: base and exp
we need specify the accepted types for each one:

```c
types = array_new(SIType, 2);
array_append(types, (SI_NUMERIC | T_NULL));
array_append(types, (SI_NUMERIC | T_NULL));
```

Reply:
But if you'll look at other functions which take 2 arguments they also have one type in their array:
	types = array_new(SIType, 1);
	array_append(types, (SI_NUMERIC | T_NULL));
	func_desc = AR_FuncDescNew("mul", AR_MUL, 2, 2, types, true, false);
	AR_RegFunc(func_desc);

Reply:
please see `_AR_EXP_ValidateInvocation`
It might be a mistake and we need to modify these functions aswell.

Reply:
After looking at _AR_EXP_ValidateInvocation:
When the number of items in the array is smaller than the number of arguments it will use the last item in the array as the expected type.
So the logic here is correct.